### ISSUE TITLE
Data Explorer: Support Copy Column Name

### ISSUE THREAD

**Issue by petetronic** [posit-dev/positron#5042]
From Discussion https://github.com/posit-dev/positron/discussions/5041
> I've been using polars a lot with Positron and one of the the annoying things is that it requires a lot of column name typing. It would be great if the Positron data viewer let you copy the column name from the column name list.

**Comment by petetronic** [comment]
We should review both the idea of a context menu option, and a keyboard shortcut, when a column summary or column header has focus to copy the column name.

**Comment by tylermorganwall** [comment]
Just wanted to 2nd this request: I've been bumping up my use of the Data Explorer and I assumed that "copy column" would give me a name, rather than the values, and was rather surprised at the result. For any sizeable table, there's not much you can do with a giant newline separated list of copied values, except for maybe copy them into Excel. The name, on the other hand, is the actual entity I'm dealing with in a data pipeline.

**Comment by Bpiep** [comment]
I would really appreciate this feature as well!

I frequently work with large datasets in R, often with many columns, and being able to copy a column name directly from the Data Explorer would be a very useful time-saver. In particular, I would like to be able to right-click a column name and copy just the column name to the clipboard, without having to copy the entire column or type the name manually.

This would be especially helpful when working with long or complex column names.

**Comment by nathanckuhn** [comment]
Agreed! In the Data Explorer, it would be nice to Right Click on the column name -- in the summary panel OR at the top of the data table column -- and have an option to "Copy Column Name". Yes, I can run "names(desired_df)" in console and then search for the column name, but doing so through Data Explorer would be significantly helpful.
